.circleci/config.yml

```diff
@@ -1,17 +1,17 @@
 version: 2
 
 defaults: &defaults
-  working_directory: ~/dd-trace-java
+  working_directory: ~/signalfx-tracing-java
```

Comment:
Shouldn't this be `signalfx-java-tracing` to match the repository name (everywhere)?